src/csharp/Microsoft.Spark.Worker.UnitTest/CommandExecutorTests.cs

```diff
@@ -309,11 +309,7 @@ public async Task TestDataFrameSqlCommandExecutorWithSingleCommand()
             var udfWrapper = new Sql.DataFrameUdfWrapper<ArrowStringDataFrameColumn, ArrowStringDataFrameColumn>(
                 (strings) =>
                 {
-                    var stringArray = (StringArray)ToArrowArray(
-                        Enumerable.Range(0, (int)strings.Length)
-                            .Select(i => $"udf: {strings[i]}")
-                            .ToArray());
-                    return ToArrowStringDataFrameColumn(stringArray);
+                    return strings.Apply(cur => $"udf: {cur}");
```

Comment:
just `(strings) => strings.Apply(cur => $"udf: {cur}")`?

Reply:
Yup. Fixed in other places too.